parser/prism/Translator.cc

```diff
@@ -1913,6 +1911,20 @@ core::NameRef Translator::translateConstantName(pm_constant_id_t constant_id) {
     return ctx.state.enterNameUTF8(parser.resolveConstant(constant_id));
 }
 
+core::NameRef Translator::nextUniqueParserName(core::NameRef original) {
+    auto nextId = *parserUniqueCounter + 1;
+    *parserUniqueCounter = nextId;
```

Comment:
Could have been:

```c++
auto nextId = *parserUniqueCounter += 1;
```

Or

```c++
auto nextId = ++*parserUniqueCounter;
```

But the precedence is non-obvious to human readers. Opting for clarity.

(And `*x++` doesn't work, that tries to increment the pointer, not the counter itself)